Scripts/Add New Borrowers.sql

```diff
@@ -0,0 +1,24 @@
+-- 5. Stored Procedure
+-- Procedure Name: sp_AddNewBorrower
+CREATE OR ALTER PROCEDURE sp_AddNewBorrower
```

Comment:
NIT: May delete it at first then create

Reply:
I added " ALTER " if  "sp_AddNewBorrower" EXISTS modify It is similar to create , true?